src/wizard/__tests__/wizard.test.tsx

```diff
@@ -78,6 +80,12 @@ describe('i18nStrings', () => {
         i18nStrings,
         steps: DEFAULT_STEPS,
       });
+
+      expect(wrapper.findByClassName(styles.navigation)!.getElement()).toHaveAttribute(
+        'aria-label',
```

Comment:
I wonder why it was not caught by AXE. Can we make some better generic rule better to test the nav elements does have the ARIA label/labelledby? We can include it to our `toValidateA11y` check as a custom rule to AXE, htmlValidator or a separate check.

Reply:
ok, got it